apps/files_sharing/css/public.scss

```
@@ -51,6 +51,18 @@ thead {
 	left: 0 !important;
 }
 
+// blur page when focus is removed
+.blur_body { 
+	filter: blur(5px); 
+}
```

Some transition to make it smoother?

```suggestion
body {
	transition: filter 0.15s;

	&.blur_body { 
		filter: blur(5px); 
	}
}
```